icu4c/source/test/intltest/calcasts.cpp

```diff
@@ -499,5 +501,81 @@ void CalendarCaseTest::Ethiopic() {
     delete c;
 }
 
+void CalendarCaseTest::Tibetan() {
+
+    //Testing Phugpa calculations 
+
+    static const TestCase tests1[] = {
+
+    // A huge list of test cases to make sure that computeTime and computeFields
+    // work properly for a wide range of data in the Tibetan calendar.
+    //
+    // Julian Day   Era Year     Month         Day  WkDay Hour Min Sec
```

Comment:
Could you cite the ref source of the following test data if you get it from somewhere. That will help us to debug in the future. thanks.  URL or Book name w/ page number would be fine.

Reply:
it's from http://kalacakra.org/calendar/tiblist.htm